src/reducers/entities/posts.js

```diff
@@ -606,7 +606,7 @@ export function reactions(state = {}, action) {
 }
 
 function storeReactionsForPost(state, post) {
-    if (!post.metadata) {
+    if (!post.metadata || !post.metadata.reactions) {
```

Comment:
Does this work if you remove the last reaction from the post? I think we use `omitempty` for this field, so it might not update to remove the last reaction

Reply:
I'm pretty sure I tried that but I'll try again really quick

Reply:
Seems to work fine 👍